:bug: MacOS-devel fails remote install

## :books: What?

**Error message:**

```
[162](https://github.com/serkor1/ta-lib-R/actions/runs/27029231056/job/79777070817#step:8:4174)
  using C compiler: ‘Apple clang version 17.0.0 (clang-1700.0.13.5)’
  using SDK: ‘MacOSX15.5.sdk’
  clang -arch arm64 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/opt/R/arm64/include   -fvisibility=hidden    -fPIC  -falign-functions=64 -Wall -g -O2  -Wall -pedantic -c assign.c -o assign.o
  In file included from assign.c:1:
  In file included from ./data.table.h:28:
  ./po.h:2:10: fatal error: 'libintl.h' file not found
      2 | #include <libintl.h>
        |          ^~~~~~~~~~~
  1 error generated.
  make: *** [assign.o] Error 1
  ERROR: compilation failed for package ‘data.table’
  * removing ‘/private/var/folders/8d/778wjbv96mq1760tv6gk374m0000gn/T/RtmpbGVQJT/pkg-libe4652973915/data.table’
  ---
  Backtrace:
  1. pak::lockfile_install(".github/pkg.lock")
  2. pak:::remote(function(...) { …
  3. err$throw(res$error)
  ---
  Subprocess backtrace:
   1. base::withCallingHandlers(cli_message = function(msg) { …
   2. get("lockfile_install_internal", asNamespace("pak"))(...)
   3. plan$install()
   4. pkgdepends::install_package_plan(plan, lib = private$library, num_workers = nw, …
   5. base::withCallingHandlers({ …
   6. pkgdepends:::handle_events(state, events)
   7. pkgdepends:::handle_event(state, i)
   8. pkgdepends:::stop_task(state, worker)
   9. pkgdepends:::stop_task_build(state, worker)
  10. base::throw(pkg_error("Failed to build source package {.pkg {pkg}}.", …
  11. | base::signalCondition(cond)
  12. global (function (e) …
  Execution halted
```

### :clipboard: TODO

- [x] Clean caches and re-run
- [ ] Install it (See: https://stackoverflow.com/questions/11370684/what-is-libintl-h-and-where-can-i-get-it)

--- a/.github/workflows/remote-install.yaml
+++ b/.github/workflows/remote-install.yaml
@@ -34,19 +34,19 @@ jobs:
       matrix:
         config:
           ## macOS
-          - {os: macos-latest,   r: 'devel', http-user-agent: 'release'}
           - {os: macos-latest,   r: 'release'}
           - {os: macos-latest,   r: 'oldrel-1'}
+          - {os: macos-latest,   r: 'oldrel-2'}
 
           ## windows
-          - {os: windows-latest, r: 'devel', http-user-agent: 'release'}
           - {os: windows-latest, r: 'release'}
           - {os: windows-latest, r: 'oldrel-1'}
+          - {os: windows-latest, r: 'oldrel-2'}
 
           # Ubuntu
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
           - {os: ubuntu-latest,   r: 'release'}
           - {os: ubuntu-latest,   r: 'oldrel-1'}
+          - {os: ubuntu-latest,   r: 'oldrel-2'}
 
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
@@ -92,19 +92,19 @@ jobs:
       matrix:
         config:
           ## macOS
-          - {os: macos-latest,   r: 'devel', http-user-agent: 'release'}
           - {os: macos-latest,   r: 'release'}
           - {os: macos-latest,   r: 'oldrel-1'}
+          - {os: macos-latest,   r: 'oldrel-2'}
 
           ## windows
-          - {os: windows-latest, r: 'devel', http-user-agent: 'release'}
           - {os: windows-latest, r: 'release'}
           - {os: windows-latest, r: 'oldrel-1'}
+          - {os: windows-latest, r: 'oldrel-2'}
 
           # Ubuntu
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
           - {os: ubuntu-latest,   r: 'release'}
           - {os: ubuntu-latest,   r: 'oldrel-1'}
+          - {os: ubuntu-latest,   r: 'oldrel-2'}
 
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}

